src/docker/compose.dss-core.yaml

```diff
@@ -19,6 +19,20 @@ services:
       APTOS_NETWORK: ${APTOS_NETWORK}
       DATABASE_URL: "postgres://econia:econia@postgres:5432/econia"
 
+  grafana:
```

Comment:
How about calling this something like `grafana_annotations`?

That way we can have another service that also just runs the Grafana image